faces/4.0/_index.md

```diff
@@ -1,56 +1,66 @@
+* [Issue #1582](https://github.com/eclipse-ee4j/faces-api/issues/1582): New annotation literals for all @Qualifiers
+* [Issue #1558](https://github.com/eclipse-ee4j/faces-api/issues/1558): Make UIComponent#subscribeToEvent() more convenient
+* [Issue #1565](https://github.com/eclipse-ee4j/faces-api/issues/1565): Skip type attribute from &lt;link> and &lt;script> when doctype is HTML5
+* [Issue #1567](https://github.com/eclipse-ee4j/faces-api/issues/1567): Improve &lt;f:ajax> behavior in composite components
+
+Including the following backwards incompatible changes:
+
+* [Issue #1552](https://github.com/eclipse-ee4j/faces-api/issues/1552): Rename "JSF" to "Faces" over all place
+* [Issue #1553](https://github.com/eclipse-ee4j/faces-api/issues/1553): Rename "http://xmlns.jcp.org/jsf/*" URL to "jakarta.faces.*" URN
+* [Issue #1546](https://github.com/eclipse-ee4j/faces-api/issues/1546): Remove all JSP support
+* [Issue #1547](https://github.com/eclipse-ee4j/faces-api/issues/1547): Remove native Managed Beans (@ManagedBean and related)
+* [Issue #1548](https://github.com/eclipse-ee4j/faces-api/issues/1548): Remove MethodBinding, ValueBinding and friends
+* [Issue #1571](https://github.com/eclipse-ee4j/faces-api/issues/1571): Remove CURRENT_COMPONENT constants from UIComponent class
+* [Issue #1578](https://github.com/eclipse-ee4j/faces-api/issues/1578): Remove deprecated methods of StateManager class
+* [Issue #1583](https://github.com/eclipse-ee4j/faces-api/issues/1583): Remove entire ResourceResolver class
+
+This release requires Java SE 11 or newer (aligned with Jakarta EE 10).
 
 * [Jakarta Faces 4.0 Release Record](https://projects.eclipse.org/projects/ee4j.faces/releases/4.0)
+    * [Jakarta EE Platform 10 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee10/JakartaEE10#jakarta-ee-10-release-plan)
+* [Jakarta Faces 4.0 Specification Document](./jakarta-faces-4.0.pdf) (PDF)
+* [Jakarta Faces 4.0 Specification Document](./jakarta-faces-4.0.html) (HTML)
+* [Jakarta Faces 4.0 Javadoc](./apidocs)
+* [Jakarta Faces 4.0 Jsdoc](./jsdoc)
+* [Jakarta Faces 4.0 Renderkitdoc](./renderkitdoc)
+* [Jakarta Faces 4.0 VDLDoc](./vdldoc)
+* [Jakarta Faces 4.0 TCK](https://download.eclipse.org/jakartaee/faces/4.0/jakarta-faces-tck-4.0.0.zip)([sig](https://download.eclipse.org/jakartaee/faces/4.0/jakarta-faces-tck-3.0.0.zip.sig),[sha](https://download.eclipse.org/jakartaee/faces/4.0/jakarta-faces-tck-4.0.0.zip.sha256),[pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
+* Maven coordinates
+  * [jakarta.faces:jakarta.faces-api:jar:4.0.0](https://search.maven.org/artifact/jakarta.faces/jakarta.faces-api/4.0.0/jar)
+
+
+# Compatible Implementations
+
+* [Mojarra 4.0.0-M5](https://github.com/eclipse-ee4j/mojarra/releases/download/4.0.0-M5-RELEASE/jakarta.faces-4.0.0-M5.jar)
```

Comment:
Should this not align with the version on the CCR?
```suggestion
* [Mojarra 4.0.0-M7](https://github.com/eclipse-ee4j/mojarra/releases/download/4.0.0-M7-RELEASE/jakarta.faces-4.0.0-M7.jar)
```

Reply:
Probably best indeed. From memory they both passed, but let's stay consistent. Thanks!